src/pages/Transaction/index.tsx

```diff
@@ -1,29 +1,45 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import { useParams } from 'react-router-dom'
 import { useQuery } from '@tanstack/react-query'
+import { ResultFormatter } from '@ckb-lumos/rpc'
 import Content from '../../components/Content'
 import { TransactionDiv as TransactionPanel } from './TransactionComp/styled'
 import { explorerService } from '../../services/ExplorerService'
 import { QueryResult } from '../../components/QueryResult'
 import { defaultTransactionInfo } from './state'
 import { useSearchParams } from '../../hooks'
+import { useCKBNode } from '../../hooks/useCKBNode'
 import { LayoutLiteProfessional } from '../../constants/common'
 import { TransactionCompLite } from './TransactionComp/TransactionLite/TransactionLite'
 import { TransactionComp } from './TransactionComp/TransactionComp'
+import { NodeTransactionComp } from './TransactionComp/NodeTransactionComp'
 import { TransactionOverviewCard } from './TransactionComp/TransactionOverview'
+import { NodeTransactionOverviewCard } from './TransactionComp/NodeTransactionOverview'
 import { TransactionDetailsHeader } from './TransactionComp/TransactionDetailsHeader'
 import { RGBDigestComp } from './TransactionComp/RGBDigestComp'
 
 export default () => {
   const { Professional, Lite } = LayoutLiteProfessional
   const { hash: txHash } = useParams<{ hash: string }>()
+  const { nodeService, activate: nodeActivate } = useCKBNode()
 
-  const query = useQuery(['transaction', txHash], async () => {
-    const transaction = await explorerService.api.fetchTransactionByHash(txHash)
-    // TODO: When will displayOutputs be empty? Its type description indicates that it will not be empty.
-    if (transaction.displayOutputs && transaction.displayOutputs.length > 0) {
-      transaction.displayOutputs[0].isGenesisOutput = transaction.blockNumber === 0
-    }
-    return transaction
+  const query = useQuery(
+    ['transaction', txHash],
+    async () => {
+      const transaction = await explorerService.api.fetchTransactionByHash(txHash)
+      // TODO: When will displayOutputs be empty? Its type description indicates that it will not be empty.
```

Comment:
The `outputs` will be empty when the transaction is pending/being processed in the backend

<img width="1271" alt="image" src="https://github.com/Magickbase/ckb-explorer-frontend/assets/7271329/b1d4bad1-a11e-416c-af91-56deb4f673c6">